src/convertToObject.js

```diff
@@ -12,7 +12,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleArr = sourceString.split(';');
+  const filterArr = styleArr.filter(arr => arr.length > 4);
+  const resObj = {};
+
+  for (let i = 0; i < filterArr.length; i++) {
```

Comment:
```suggestion
  for ([key, value] of filterArr) {
```
а тут после `map` через деструктуризацию сразу доставать то что нужно